pyquil/api/_quantum_computer.py

```diff
@@ -14,6 +14,8 @@
 #    limitations under the License.
 ##############################################################################
 import re
+import socket
+import sys
```

Comment:
I think you can drop this `sys` import now, no?

Reply:
Good catch!